roles/bootstrap-os/files/bootstrap.sh

```diff
@@ -18,7 +18,7 @@ mv -n pypy-$PYPY_VERSION-linux64 pypy
 
 ## library fixup
 mkdir -p pypy/lib
-ln -snf /lib64/libncurses.so.5.9 $BINDIR/pypy/lib/libtinfo.so.5
+ln -snf /lib64/libncurses.so.6.1 $BINDIR/pypy/lib/libtinfo.so.5
```

Comment:
Are we sure this actually exists?

Reply:
I am sure, as it did not work without this change for me and does so with. I linked to an issue where you can find more details about it.